include/openssl/ssl.h

```diff
@@ -217,11 +217,9 @@ typedef struct srtp_protection_profile_st {
 
 DEFINE_STACK_OF(SRTP_PROTECTION_PROFILE)
 
-typedef int (*tls_session_ticket_ext_cb_fn) (SSL *s,
-                                             const unsigned char *data,
+typedef int (*tls_session_ticket_ext_cb_fn) (SSL *s, const unsigned char *data,
                                              int len, void *arg);
```

Comment:
Huh, the old form is surprising.  I could understand breaking early to get 'data' and 'len' on the same line, but the old form seems to have not had any real benefit.